tests/unit/challenges/test_views.py

```diff
@@ -6020,14 +6020,13 @@ def test_create_challenge_using_github_without_branch_name(self):
             self.assertEqual(response.status_code, 201)
             self.assertEqual(response.json(), expected)
 
-        # Verify github_branch defaults to empty string when not provided
+        # Verify github_branch defaults to "challenge" when not provided
```

Comment:
Please also add when the branch name is "challenge", when branch name is "xyzabc", and when branch name is "challenge-2025-v2" (whatever format you have chosen).